mumbleBot.py

```diff
@@ -459,6 +516,20 @@ def _download(self, item):
     def loop(self):
         while not self.exit and self.mumble.is_alive():
 
+            # Release started tasks if any
```

Comment:
What do you think of moving this part to the place between L516 and 517, outside the while loop?

Reply:
That would be better as there is no active monitoring by the bot.